suzieq/engines/pandas/path.py

```diff
@@ -422,23 +495,21 @@ def _get_nexthops(self, device: str, vrf: str, dest: str, is_l2: bool,
         if not rslt.empty and (len(rslt.nexthopIps.iloc[0]) != 0 and
                                rslt.nexthopIps.iloc[0][0] != '') and (
                                    rslt.protocol.iloc[0] != 'hmm'):
+            if not rslt.empty and (len(rslt.nexthopIps.iloc[0]) != 0 and
```

Comment:
We are checking two times the following statement: `not rslt.empty and (len(rslt.nexthopIps.iloc[0]) != 0 and rslt.nexthopIps.iloc[0][0] != '')`. Why not merging the two if statements?